README.md

```diff
@@ -91,7 +91,8 @@ Currently, pods request resource requirements are considered for computing node
 There is another configurable threshold, `targetThresholds`, that is used to compute those potential nodes
 from where pods could be evicted. Any node, between the thresholds, `thresholds` and `targetThresholds` is
 considered appropriately utilized and is not considered for eviction. The threshold, `targetThresholds`,
-can be configured for cpu, memory, and number of pods too in terms of percentage.
+can be configured for cpu, memory, and number of pods too in terms of percentage. Notice that `pods` must be
+configured and if `cpu` or `memory` are not configured, they'll be set to default value `100`.
```

Comment:
why is `pods` required now, and we should add some validation to ensure that it's set

Reply:
I only added default value to `cpu` and `mem` and didn't change the validate func .Current `validateTargetThresholds` func only checks wheather `pods` is configured, if not, `LowNodeUtilization` will stop, so only `pods` is required. Should we treat`pods` as same as `cpu` and `mem`, and `validateTargetThresholds` func returns true if any of these resources configured?

Reply:
Maybe we can use `validateThresholds` to check `targetThresholds`

Reply:
It looks like you pushed a change to set a default for `pods`, is that correct? If so please remember to update this readme note

Reply:
Yes, I'll update it in later commit